src/main/resources/db/changelog/changes/06-create-category-table.yaml

```
@@ -0,0 +1,30 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-categories-table
+      author: root
+      changes:
+        - createTable:
+            tableName: categories
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: VARCHAR(255)
+                  constraints:
+                    unique: true
+                    nullable: false
+              - column:
+                  name: description
+                  type: VARCHAR(2000)
```

```suggestion
                  type: VARCHAR(255)
```
2000 - it's too big size.